propolis-server/rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly
```

Comment:
Not actually using USDT in the server yet, but this seems like a prereq

Reply:
It's used in `propolis` the library, so it is a prereq